core/lib/src/response/responder.rs

```diff
@@ -98,6 +98,12 @@ use crate::request::Request;
 ///     to the client. If the `Result` is `Err`, the wrapped `Err` responder is
 ///     used to respond to the client.
 ///
+///   * **::either::Either&lt;L, R>** _where_ **L: Responder** and **R:
```

Comment:
Why not just call it `Either`? That is, why use `either::Either`?

Reply:
In this case because it's external unlike the others. Given that you suggested re-exporting it, that reasoning no longer applies.